preprocessors/openstreetmap/osm_service.py

```diff
@@ -67,6 +67,7 @@ def server_config2(url, bbox_coord):
     (node({lat_min},{lon_min},{lat_max},{lon_max}) ["amenity"];
     way({lat_min},{lon_min},{lat_max},{lon_max}) ["amenity"];
     rel({lat_min},{lon_min},{lat_max},{lon_max}) ["amenity"];
+    way({lat_min},{lon_min},{lat_max},{lon_max}) ["building"];
```

Comment:
This line queries other features (POIs) that may not be available under the ```amenity``` key.

Reply:
Could you give an example of one such feature?

Reply:
For an example, hotels are supposed to be under the ```amenity``` key, but this was only found under the ```building``` key. There are many others like that, depending on the queried area.
```{
				"id": 341125395,
				"lat": 45.5139393,
				"lon": -73.557474,
				"name": "Hôtel Le Roberval",
				"cat": "building",
				"tourism": "hotel"
			},```
[Full file attached](https://github.com/Shared-Reality-Lab/IMAGE-server/files/11525773/fullfile.txt) using with the request below:
```
{
  "request_uuid": "a3d7b6be-8f3b-47ba-8b17-4b0557e7a8ce",
  "timestamp": 1637789517,
	"coordinates": {
     "latitude": 45.5140079,	
     "longitude":  -73.5576234
   },
  "context": "",
  "language": "en",
  "url": "https://fake.site.com/some-url",
  "capabilities": ["ca.mcgill.a11y.image.capability.DebugMode"],
  "renderers": [
    "ca.mcgill.a11y.image.renderer.Text",
    "ca.mcgill.a11y.image.renderer.SimpleAudio",
    "ca.mcgill.a11y.image.renderer.SegmentAudio",
		"ca.mcgill.a11y.image.renderer.SVGLayers"
  ]
}
```

Reply:
For an example, hotels are supposed to be under the ```amenity``` key, but this was only found under the ```building``` key. There are many others like that, depending on the queried area.
```{
				"id": 341125395,
				"lat": 45.5139393,
				"lon": -73.557474,
				"name": "Hôtel Le Roberval",
				"cat": "building",
				"tourism": "hotel"
			},
```
[Full file attached](https://github.com/Shared-Reality-Lab/IMAGE-server/files/11525773/fullfile.txt) using with the request below:
```
{
  "request_uuid": "a3d7b6be-8f3b-47ba-8b17-4b0557e7a8ce",
  "timestamp": 1637789517,
	"coordinates": {
     "latitude": 45.5140079,	
     "longitude":  -73.5576234
   },
  "context": "",
  "language": "en",
  "url": "https://fake.site.com/some-url",
  "capabilities": ["ca.mcgill.a11y.image.capability.DebugMode"],
  "renderers": [
    "ca.mcgill.a11y.image.renderer.Text",
    "ca.mcgill.a11y.image.renderer.SimpleAudio",
    "ca.mcgill.a11y.image.renderer.SegmentAudio",
		"ca.mcgill.a11y.image.renderer.SVGLayers"
  ]
}
```